frontend/cypress/e2e/Pages/User/UserCreation.test.js

```diff
@@ -0,0 +1,88 @@
+import {
+  Given,
+  When,
+  Then,
+  After
+} from '@badeball/cypress-cucumber-preprocessor'
+
+Given('the IDIR user is on the login page', () => {
+  cy.clearAllCookies()
+  cy.clearAllLocalStorage()
+  cy.clearAllSessionStorage()
+  cy.visit('/')
+  cy.getByDataTest('login-container').should('exist')
+})
+
+When('the IDIR user logs in with valid credentials', () => {
+  cy.loginWith(
+    'idir',
+    Cypress.env('admin_idir_username'),
+    Cypress.env('admin_idir_password')
+  )
+  cy.wait(5000)
+  cy.setIDIRRoles('analyst')
+  cy.visit('/')
+  cy.getByDataTest('dashboard-container').should('exist')
+})
+
+When('the IDIR user navigates to the user creation page', () => {
+    cy.get('a[href="/admin"]').click()
+    cy.wait(3000)
```

Comment:
suggestion: Can we remove these waits to speed up the test?
Most of the cypress commands (url and contains) should already be re-trying until the timeout is reached or it fails

Reply:
Of course!